### PR TITLE
Adding js support in lambda judge

### PR DESCRIPTION
**Pull request by edogrigqv2**

### CODE REVIEW

File: coderunners/compilers.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from abc import abstractmethod, ABC
 from dataclasses import dataclass
+from http.server import executable
```
Comment (MartinXPN) [inline coderunners/compilers.py:3]: Do we need this?

**Review by MartinXPN — COMMENTED** [review]
Looks good 👍 just one question regarding the import

File: coderunners/compilers.py (hunk 1 above)
Comment (edogrigqv2) [reply coderunners/compilers.py]: Fixed

**Review by MartinXPN — APPROVED** [review]
LGTM

**Pull request merged**